WordPress/Classes/ViewRelated/Me/All Domains/Views/AllDomainsListViewController.swift

```diff
@@ -118,6 +120,25 @@ final class AllDomainsListViewController: UIViewController {
         ])
     }
 
+    /// Force the navigation bar separator to be always visible.
+    private func setupNavigationBarAppearance() {
+        let appearance = self.navigationController?.navigationBar.standardAppearance
+        self.navigationItem.scrollEdgeAppearance = appearance
+        self.navigationItem.compactScrollEdgeAppearance = appearance
+    }
+
+    private func setupRefreshContro() {
+        let action = UIAction { [weak self] action in
+            guard let self, let refreshControl = action.sender as? UIRefreshControl else {
+                return
+            }
+            self.tableView.sendSubviewToBack(refreshControl)
```

Comment:
This fixes an issue where the `UIRefreshControl` might appear above the table view cells.